nexus/proposals/api.py

```diff
@@ -17,11 +17,16 @@ class ProposalViewSet(mixins.ListModelMixin,
                       mixins.DestroyModelMixin,
                       viewsets.GenericViewSet):
     queryset = models.Proposal.objects.all()
-    serializer_class = serializers.ProposalSerializer
+
+    def get_serializer_class(self):
+        if self.action in ['accept', 'retract']:
+            return serializers.StatusUpdateSerializer
```

Comment:
What is `StatusUpdateSerializer`?

Reply:
And this comes in use when Proposal status is to be updated. 
https://github.com/CuriousLearner/nexus/pull/11/files#diff-1b42a777daae5eec2c8eb789a4b1f06eR29

Reply:
Please name things appropriately.

Reply:
ACK